### PR TITLE
send pdf base64 for validation and show in confirmation step

### PR DESCRIPTION
**Pull request by rfreitas**
1. Files to upload are converted to base64 and sent for validation on the server
2. In the confirmation step they are shown as images (if possible) or in an embed object (if a pdf for example)

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://95-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://95-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://95-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/utils/file.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import loadImage from 'blueimp-load-image/js/load-image'
+import {canvasToBase64Images, toLossyImageDataUrl} from './canvas.js'
+
+export const fileToBase64 = (file, callback, errorCallback) => {
+  const reader = new FileReader();
+  reader.readAsDataURL(file);
+  reader.onload = function () {
+    //console.log(reader.result);
```
Comment (stephencookdev) [inline src/components/utils/file.js]: Accidental comment here

File: src/components/utils/file.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  reader.onerror = function (error) {
+    console.log('File Reading Error: ', error);
+    errorCallback(error)
+  };
+}
+
+export const fileType = file => file.type.split('/')[1]
+
+export const isOfFileType = (fileTypeList, file) =>
+        fileTypeList.some(acceptableFileType =>
```
Comment (stephencookdev) [inline src/components/utils/file.js]: Your spacing seems a little off here

**Comment by rfreitas** [comment]
Travis automatic deployment: https://95-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**